packages/@okta/i18n/src/properties/login.properties

```diff
@@ -1299,12 +1299,18 @@ oie.return.to.original.tab = Close this window anytime.
 idx.operation.cancelled.by.user = Operation cancelled by user.
 idx.transferred.to.new.tab = Flow continued in a new tab.
 idx.return.link.expired = This email link has expired. To resend it, return to the screen where you requested it.
-idx.return.link.otponly.title = Return to the location where you initiated the verification process and enter this one-time password.
+idx.return.link.otponly.your.verification.code = Your verification code
+# {0} is the name of the flow that initialized the request: sign in, password reset, account unlock, registration
+idx.return.link.otponly.enter.code.on.page = Enter this code on the {0} page.
+idx.return.link.otponly.enter.code.on.page.sign.in = sign in
+idx.return.link.otponly.enter.code.on.page.password.reset = password reset
+idx.return.link.otponly.enter.code.on.page.account.unlock = account unlock
+idx.return.link.otponly.enter.code.on.page.registration = sign up
 idx.return.link.otponly.request = Request from:
-idx.return.link.otponly.warning = If you didn’t request this verification code, you can safely ignore this message. Your account isn’t compromised and can’t be accessed without this code.
+idx.return.link.otponly.warning.text = If you didn’t request this code, you can ignore this message. Your account is safe and can only be accessed with this code.
 idx.enter.otp.in.original.tab = Enter the OTP in your original authentication location.
 # {0} is the name of the app trying to be accessed with this email magic link login
-idx.return.link.otponly.accessing.app = Accessing {0}
+idx.return.link.otponly.app = {0}
```

Comment:
I think it would be good to get @gretchen-okta 's review on these strings

Reply:
Thank you Justin, the texts come from our designer Maggie Cai.

Reply:
Connected with @gretchen-okta offline, updated this based on feedback.